vendor/github.com/google/gopacket/layers/bitfield.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 The GoPacket Authors. All rights reserved.
```

Comment:
are these changes from gopacket's recent changes?

Reply:
Yes.

Reply:
Ok, best way for such PR is to split commits into two commits. One for the external changes and one for changes you made. It's easier to review that way.

Also please add some info on how this PR was verified.

Reply:
All the changes are made by "Make verify-vendor" after running "go get gopacket@commitid".

Reply:
got it.